chrome/content/zotero/preferences/quickCopySiteEditor.xul

```diff
@@ -46,6 +47,7 @@
 				var io = window.arguments[0];
 				io.domain = document.getElementById('zotero-quickCopy-domain').value;
 				io.format = document.getElementById('zotero-quickCopy-menu').value;
+				io.locale = document.getElementById('locale-menu').label !== '' ? document.getElementById('locale-menu').value : '';
```

Comment:
Right. Maybe I thought it better to clearly separate the blank label (which is more of a visual thing) from the fallback locale, even though they're both an empty string. Don't really remember.